basket/base/management/commands/rqworker.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from basket.base.rq import get_worker
+
+
```

Comment:
Yes, I chose to simplify and not use `django_rq` since it adds a lot of indirection I wasn't too crazy about. But looking at rq's [`rqworker`](https://github.com/rq/rq/blob/master/rq/cli/cli.py#L198) CLI makes me curious if I need a bit more in here.

Reply:
I did review the `cli/rqworker` and I think we're good. We're picking up a lot of the same defaults just by using our `get_worker` and default kwargs stuff.